### DISCUSSION TITLE
Introduce blueprint mode for construction

### DISCUSSION THREAD

**Discussion by Miszkaaaa** [voxeltycoon/issues#780] [General] [answered]
Description:
Introduce a switch for buidlings/roads/railroad so it is possible to make a project of a construct and then materialize it when the company has the money to do it or the player wants so

Full description and concept:

The buildings/roads/rail etc. could have a "blueprint mode" switch. Then a building would appear with a blue outline - it could be placed and cost only some % (like 10%)  as a fee for "reserving" the land. After placing it - the player would have another tool to "materialize it".

This would allow more planning and better managment of the real estate

![image](https://user-images.githubusercontent.com/10045272/118547986-0abc8a00-b75a-11eb-9676-e96c3f46d323.png)

**Comment by victorleblais** [comment, marked as answer]
Planning mode and blueprints are features on the roadmap !
https://voxeltycoon.xyz/roadmap

**Reply by Miszkaaaa** [reply]
oki doki, is it possible to put a link to the roadmap in this repo (in the readme.md?) so the people don't add existing planned features?
